src/components/YourCompany/OfficeAddresses.vue

```diff
@@ -395,17 +398,20 @@ export default class OfficeAddresses extends Mixins(CommonMixin, EntityFilterMix
   }
 
   /**
-   * Sets local address data and "inherited" flags from store.
+   * Sets local address data and "inherit" flags from store.
    */
   private setLocalProperties (): void {
     if (this.getOfficeAddresses.registeredOffice) {
       this.mailingAddress = { ...this.getOfficeAddresses.registeredOffice.mailingAddress }
       this.deliveryAddress = { ...this.getOfficeAddresses.registeredOffice.deliveryAddress }
 
-      // compare addresses to set the "inherited mailing" flag
+      // compare addresses to set the "inherit mailing" flag
+      // ignore Address Type since it's different
+      // ignore Address Country Description since it's not always present
```

Comment:
I don't know where Address Country Description comes from, but sometimes it's there and sometimes it isn't, so just ignore it in the comparisons.